app/build.gradle

```diff
@@ -190,6 +193,7 @@ dependencies {
 
     kapt "com.jakewharton:butterknife-compiler:${deps.butterknife}"
     kapt "org.greenrobot:eventbus-annotation-processor:${deps.eventbus}"
+    compile project(path: ':ui:tutorial-renderer')
```

Comment:
you shouldn't need this since you are already including it as an `implementation` dependency

Reply:
this should be unnecessary since you already specify the dependency on line 149

Reply:
This still needs to be removed